src/components/DateRangeInput.tsx

```diff
@@ -20,7 +20,7 @@ export default function DateRangeInput({
 }: DateRangeInputProps): JSX.Element {
   const now = DateTime.now();
   const shortcuts: { [key: string]: { text: string, period: { start: Date, end: Date } } } = {
-    today: {
+    t: {
```

Comment:
apparently having the `today` key has some magic behavior for the datepicker where it ignores the date you pass for setting the calendar.